readme.md

```diff
@@ -24,7 +24,9 @@ Second, you can call the calculate function by passing the following information
 - `page`: The current page
 - `size`: The amount of items shown per page
 - `total`: The amount of total items in the list/collection
-- `maxPages`: The maximum amount of pages that should be shown (default: 5)
+- `config`: A configuration object containing the following possible configuration values:
+  - `firstPage`: The first page of the pagination (default: 1)
+  - `maxPages`: The maximum amount of pages that should be shown (default: 5)
```

Comment:
Are you sure you want to have a default for maxPages? How would I say that I want all pages to be shown?

Reply:
This is the number of pages that should be shown blow. For example if you have hundreds of results, then we show only the next 2 pages. Otherwise, it breaks most of the times on mobile.

Do you know a sceznario, where you want to show all pages?

Reply:
I like the feature, but I would just keep the possibility to also show all pages. I think a lib should support that.

Reply:
Would you than also stick to this config object? Because in our case with our designs we then always have to pass this parameter.